src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -434,14 +434,13 @@ def _processPileupMixingModules(self, pileupDict, seLocalName, modules, requeste
                     if requestedPileupType == 'data':
                         baggage = self.job.getBaggage()
                         if getattr(baggage, 'skipPileupEvents', None) is not None:
+                            # For deterministic pileup, we want to shuffle the list the
+                            # same for every job in the task and skip events
+                            random.seed(self.job['task'])
```

Comment:
Sure, that will be my first attempt! The painful part is that this logging is only available in the wrapper/worker node, which is not returned in the logarchive. I'll keep you up-to-date ;-)